keyserver/coniksserver/main.go

```diff
@@ -0,0 +1,73 @@
+package main
+
+import (
+	"bytes"
+	"flag"
+	"os"
+	"os/signal"
+
+	"github.com/BurntSushi/toml"
+	"github.com/coniks-sys/coniks-go/keyserver"
+	"github.com/coniks-sys/coniks-go/utils"
+)
+
+const ConfigFile = "config.toml"
+const PoliciesFile = "policies.toml"
+
+func main() {
+	genConfigPtr := flag.Bool("genconfig", false, "Generate server config")
```

Comment:
If there was only one config file, we could pass in a path for where to generate it,

```
flag.Bool("genconfig", "config.toml", "Generate server config")
```

and get rid of the globals (`ConfigFile`, `PoliciesFile`).